deploy-manage/deploy/cloud-on-k8s/pod-disruption-budget.md

```diff
@@ -34,14 +70,15 @@ spec:
           elasticsearch.k8s.elastic.co/cluster-name: quickstart
 ```
 
-::::{note} 
+This will cause the ECK operator to only create the PodDisruptionBudget defined in the spec. It will not create any additional PodDisruptionBudgets.
+
+::::{note}
 [`maxUnavailable`](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#arbitrary-controllers-and-selectors) cannot be used with an arbitrary label selector, therefore `minAvailable` is used in this example.
 ::::
 
+#### Pod disruption budget per nodeSet [k8s-pdb-per-nodeset]
 
-## Pod disruption budget per nodeset [k8s-pdb-per-nodeset]
-
-You can specify a PDB per nodeset or node role.
+You can specify a PDB per nodeSet or node role.
```

Comment:
```suggestion
You can specify a PDB per `nodeSet` or node role.
```

I would put these into backticks to make clear this is a reference to our YAML CRD schema. Otherwise it just looks like a typo.